codewof/programming/content/en/rollercoaster-ride/solution.py

```diff
@@ -0,0 +1,7 @@
+def ride_the_rollercoaster(age, with_an_adult):
+    if age >= 10:
```

Comment:
The question says they need to be 12 or older, but this condition says 10 or older